### DISCUSSION TITLE
bring it back: hold-note from osu!stream

### DISCUSSION THREAD

**Discussion by zabergan** [ppy/osu#30288] [Ideas] [answered]
As you might know, osu!stream was a side-project from peppy for iOS and arcade machines, and it had a special hold note.

![rhythm-game-android-ios---osu-stream-gameplay_edit-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/9e11a322-dc72-4343-b71f-33ca6881d5f0)

We should have the discussion about it, if we want to have the opportunity to map with it.
I think its a fairly niche, but otherwise good tool especially for someone who's playing on slower/easier maps.

**Comment by shinmorisawa** [comment]
I agree. We should consider it.

**Comment by Joehuu** [comment, marked as answer]
This is a duplicate of https://github.com/ppy/osu/discussions/27394, please upvote that!
